### ISSUE TITLE
[NN] Plot test stats over number of epochs

### ISSUE THREAD

**Issue by michelole** [bst-mug/n2c2#101]
Hypothesis: the more epochs, the more we overfit.

**Comment by michelole** [comment]
tl;dr: fasttext with pre-trained embeddings indeed converges faster (less epochs) to the max accuracy. This effect is stronger the larger the learning rate. Anyway, fasttext with self-trained embeddings always seem to catch up at some point.
